### PR TITLE
Create payload.txt

### PR DESCRIPTION
**Pull request by mavisinator30001**

### CODE REVIEW

**Review by hak5peaks — CHANGES_REQUESTED** [review]

File: payloads/library/prank/payload.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+DELAY 1000
+TAB
+TAB
+TAB
+TAB
+TAB
+TAB
+ENTER
+DELAY 200
+HOLD DOWN
```
[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:29]: Sorry it took me so long to respond to these, I will work on those improvements! Thank you! I also used my gaming pc to test this payload, but I also realize that it is faster than a majority of other computers, so all of this feedback helps a ton! Thank you!

File: payloads/library/prank/payload.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+DELAY 2000
+ATTACKMODE HID
+CTRL GUI d
+DELAY 500
+GUI i
+DELAY 1000
+TAB
+TAB
+TAB
+TAB
```
[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:16]: I didn't read thoroughly through the Duckyscript documentation, just not how I retain information.... ADHD. That is very good to know, thank you.

File: payloads/library/prank/payload.txt [hunk 3]
```diff
@@ -0,0 +1,60 @@
+REM Title: Resolution Prank
```
[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:1]: Understood.

File: payloads/library/prank/payload.txt [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+REM Author: mavisinator30001
+REM Description: The duck will open a settings window and decrease the screen reolution until caps lock is toggled, in which case it returns to normal
+REM Target: Any Unlocked Windows 11/10 system
+
+
+DELAY 2000
```
[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:7]: What would be the difference between that and saying, for instance, `IF ($_OS == WINDOWS)` and just having the code in the IF statement ?

[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:7]: When I use the `IF ($_OS == WINDOWS)` statement and test my payload on my personal laptop, which runs Windows 11, it does not sense that it is a Windows OS and just skips the entire statement

[the remark this answers is not in the document]
Comment (mavisinator30001) [reply payloads/library/prank/payload.txt:7]: For some reason I could not get it to sense Windows from my laptop, so I will just provide a disclaimer that it only works on Windows systems.

**Pull request closed without merging**